qiskit_optimization/algorithms/admm_optimizer.py

```diff
@@ -42,7 +42,9 @@
 class ADMMParameters:
     """Defines a set of parameters for ADMM optimizer."""
 
-    def __init__(
+    # Delete all pylint checks because (too-many-positional-arguments) is unknown
+    # option value in some platforms in github CI
+    def __init__(  # pylint: disable=all
```

Comment:
`disable=all` is too broad.
How about the following?
```suggestion
    def __init__(  # pylint: disable=too-many-positional-arguments
```

Reply:
If I use `# pylint: disable=too-many-positional-arguments` in some platform tests I'm getting an error that `too-many-positional-arguments` is an unknown pylint flag. I guess this happens because some of the platforms doesn't have the updated version of the pylint.

Reply:
That happens in Python 3.8 as the newer pylint is not supported there and an older version of pylint that is compatible gets installed that does not have that new option. What we did in other applications and algorithms was to add  "unknown-option-value" to pylint config to disable the error you get in that case from this new option. For instance see this config https://github.com/qiskit-community/qiskit-algorithms/blob/8709f009b61b1c573d802b4b3f8201136a29156a/pyproject.toml#L55 though elsewhere we unified much of the config in a pyproject.toml rather than having the separate config files but I would hope it should work the same way. For instance this is the PR in Machine learning that did the equivalent (seems the change to have the config in pyproject was done in that same PR)

Reply:
Thanks @woodsp-ibm! I added both `unknown-option-value` and `too-many-positional-arguments` in the `.pylintrc` file.

Reply:
I guess Steve suggested adding only `unknown-option-value` to pylintrc and changing `pylint: disable` for each `__init__`. Could you confirm it, @woodsp-ibm? 

His link adds only `unknown-option-value` https://github.com/qiskit-community/qiskit-algorithms/blob/8709f009b61b1c573d802b4b3f8201136a29156a/pyproject.toml#L55
qiskit-algorithms has `pylint: disable=too-many-positional-arguments` for each file.
https://github.com/qiskit-community/qiskit-algorithms/blob/8709f009b61b1c573d802b4b3f8201136a29156a/qiskit_algorithms/phase_estimators/ipe.py#L62

Reply:
Yes, elsewhere we added the disable too many args on each failing line rather than disabling it globally with a view that for any new code we ought to do better so that this does not get raised. And as you say you then need the unknown option value disable to suppress the error you then get from the older pylint used in 3.8 .

Reply:
Thanks guys! I modified the code accordingly! I hope I did it correctly this time :P